builds/otel.sh

```diff
@@ -20,6 +20,6 @@ zip -u "$DESTDIR/opentelemetry-java-contrib-jmx-metrics.jar" "META-INF/LICENSE.r
 cd ../opentelemetry-operations-collector
 # Using array assignment to drop the filename from the sha256sum output
 JAR_SHA_256=($(sha256sum "$DESTDIR/opentelemetry-java-contrib-jmx-metrics.jar"))
-go build -buildvcs=false -o "$DESTDIR/otelopscol" \
+go build -tags=gpu -buildvcs=false -o "$DESTDIR/otelopscol" \
```

Comment:
Is the new tag required for the latest otel, or are we picking up a feature along the way?

Reply:
This is a refactor we're picking up. I made the adjustment after discussing with Lujie about how to handle building the GPU receivers. Here's the PR: https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/156